### PR TITLE
Add: Added python and csharp code samples in doc Mouse action in detail 

### PR DESCRIPTION
**Pull request by hiroksarker**
### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 1]
```diff
@@ -57,7 +57,34 @@ searchBtn = driver.find_element_by_link_text("Sign in")
...
+            try
+            {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Sign in"));
+                Actions actionProvider = new Actions(driver);
+                // Perform click-and-hold action on the element
+                actionProvider.ClickAndHold(searchBtn).Build().Perform();
+            }
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: Hi @hiroksarker,

Code looks way too indented to the right, can you please align it?

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 2]
```diff
@@ -130,7 +157,34 @@ searchBtn = driver.find_element_by_link_text("Sign in")
...
+            IWebDriver driver = new ChromeDriver();
+            try
+            {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Sign in"));
+                Actions actionProvider = new Actions(driver);
+                // Perform context-click action on the element
+                actionProvider.ContextClick(searchBtn).Build().Perform();
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: Same as above

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 3]
```diff
@@ -276,7 +357,35 @@ searchBtn = driver.find_element_by_link_text("Gmail")
...
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Sign in"));
+                Actions actionProvider = new Actions(driver);
+                // Performs mouse move action onto the element
+                actionProvider.MoveToElement(searchBtn).Build().Perform();
+            }
+            finally
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: same as above

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 4]
```diff
@@ -355,7 +464,37 @@ yOffset = 100
...
+                driver.Navigate().GoToUrl("https://google.com");
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Gmail"));
+                // Set x and y offset positions of element
+                int xOffset = 100;
+                int yOffset = 100;
+                Actions actionProvider = new Actions(driver);
+                // Performs mouse move action onto the offset position
+                actionProvider.MoveByOffset(xOffset, yOffset).Build().Perform();
+            }
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: same as above

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 5]
```diff
@@ -518,7 +686,38 @@ targetEleYOffset = targetEle.location.get("y")
...
+                driver.Navigate().GoToUrl("https://crossbrowsertesting.github.io/drag-and-drop");
+                // Store 'box A' as source element
+                IWebElement sourceEle = driver.FindElement(By.Id("draggable"));
+                // Store 'box B' as source element
+                IWebElement targetEle = driver.FindElement(By.Id("droppable"));
+                int targetEleXOffset = targetEle.Location.X;
+                int targetEleYOffset = targetEle.Location.Y;
+                Actions actionProvider = new Actions(driver);
+                // Performs drag and drop action of sourceEle onto the targetEle
+                actionProvider.DragAndDropToOffset(sourceEle, targetEleXOffset, targetEleYOffset).Build().Perform();
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: same as above

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md [hunk 6]
```diff
@@ -270,7 +351,35 @@ searchBtn = driver.find_element_by_link_text("Gmail")
...
+            {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Sign in"));
+                Actions actionProvider = new Actions(driver);
+                // Performs mouse move action onto the element
+                actionProvider.MoveToElement(searchBtn).Build().Perform();
+            }
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md]: same as above

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md [hunk 7]
```diff
@@ -512,7 +680,38 @@ targetEleYOffset = targetEle.location.get("y")
...
+                driver.Navigate().GoToUrl("https://crossbrowsertesting.github.io/drag-and-drop");
+                // Store 'box A' as source element
+                IWebElement sourceEle = driver.FindElement(By.Id("draggable"));
+                // Store 'box B' as source element
+                IWebElement targetEle = driver.FindElement(By.Id("droppable"));
+                int targetEleXOffset = targetEle.Location.X;
+                int targetEleYOffset = targetEle.Location.Y;
+                Actions actionProvider = new Actions(driver);
+                // Performs drag and drop action of sourceEle onto the targetEle
+                actionProvider.DragAndDropToOffset(sourceEle, targetEleXOffset, targetEleYOffset).Build().Perform();
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md]: same as above

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.es.md [hunk 8]
```diff
@@ -519,7 +687,38 @@ targetEleYOffset = targetEle.location.get("y")
...
+                driver.Navigate().GoToUrl("https://crossbrowsertesting.github.io/drag-and-drop");
+                // Store 'box A' as source element
+                IWebElement sourceEle = driver.FindElement(By.Id("draggable"));
+                // Store 'box B' as source element
+                IWebElement targetEle = driver.FindElement(By.Id("droppable"));
+                int targetEleXOffset = targetEle.Location.X;
+                int targetEleYOffset = targetEle.Location.Y;
+                Actions actionProvider = new Actions(driver);
+                // Performs drag and drop action of sourceEle onto the targetEle
+                actionProvider.DragAndDropToOffset(sourceEle, targetEleXOffset, targetEleYOffset).Build().Perform();
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.es.md]: same as above

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 9]
```diff
@@ -355,7 +464,37 @@ yOffset = 100
...
+    {
+        public static void Main(string[] agrs)
+        {
+            IWebDriver driver = new ChromeDriver();
+            try
+            {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Gmail"));
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: can you please change variable name to 'gmailLink' and also can you update the comment.

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 10]
```diff
@@ -276,7 +357,35 @@ searchBtn = driver.find_element_by_link_text("Gmail")
...
+            try
+            {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Sign in"));
+                Actions actionProvider = new Actions(driver);
+                // Performs mouse move action onto the element
+                actionProvider.MoveToElement(searchBtn).Build().Perform();
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: This code sample is different when compared to others. can you please change it

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md [hunk 11]
```diff
@@ -349,7 +458,37 @@ yOffset = 100
...
+    {
+        public static void Main(string[] agrs)
+        {
+            IWebDriver driver = new ChromeDriver();
+            try
+            {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://google.com");
+                // Store 'google search' button web element
+                IWebElement searchBtn = driver.FindElement(By.LinkText("Gmail"));
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md]: can you please change variable name to 'gmailLink' and also can you update the comment.

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md (hunk 6 above)
Comment (Harsha509) [reply docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md]: The code sample is different from others in this section

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 12]
```diff
@@ -233,7 +263,17 @@ public class moveToElement {
...
   {{< code-panel language="python" >}}
-# Please raise a PR
+from selenium import webdriver
+driver = webdriver.Chrome()
+
+# Navigate to url
+driver.get("http://www.google.com")
+
+# Store 'google search' button web element
+searchBtn = driver.find_element_by_link_text("Gmail")
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: can you please change variable name to 'gmailLink' and also can you update the comment.

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md [hunk 13]
```diff
@@ -300,7 +340,19 @@ public class moveByOffset {
...
```
Comment (Harsha509) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.de.md]: same as here

**Comment by hiroksarker** [comment]
@Harsha509 

> Hi @hiroksarker,
> 
> I've provided a few comments on this PR.

Thanks, fixed code alignment, and variable name change

**Pull request merged**